test/e2e/specs/users/invite__revoke.ts

```diff
@@ -50,13 +50,9 @@ describe( DataHelper.createSuiteTitle( `Invite: Revoke` ), function () {
 				sentTo: testEmailAddress,
 			} );
 			const links = await emailClient.getLinksFromMessage( message );
-			const acceptInviteLink = links.find( ( link: string ) =>
+			acceptInviteLink = links.find( ( link: string ) =>
 				link.includes( 'accept-invite' )
 			) as string;
-
-			expect( acceptInviteLink ).toBeDefined();
-
-			adjustedInviteLink = DataHelper.adjustInviteLink( acceptInviteLink );
```

Comment:
**question:nonblocking** - has the `adjustInviteLink` method been removed?

Reply:
adjustInviteLink hasn't been removed from data-helper.ts because I'm concerned that we're suddenly going to need it again since I don't know what change was made and/or why. If you're confident we won't need it, I can pull it out pretty fast.

Reply:
>  If you're confident we won't need it, I can pull it out pretty fast.

I don't think we'll need it again and if we suddenly need it again, reverting this PR _should_ resolve that. But it isn't a blocking requirement in my eyes.